src/source/canvas_source.js

```diff
@@ -167,10 +167,6 @@ class CanvasSource extends ImageSource {
      * @method setCoordinates
      * @instance
      * @memberof CanvasSource
-     * @param {Array<Array<number>>} coordinates Four geographical coordinates,
```

Comment:
I don't think we need to specify params for inherited methods.

Reply:
Is this information expressed anywhere else after we delete it?